### ISSUE TITLE
All patterns converted to lower-case

### ISSUE THREAD

**Issue by vinzent** [mxey/ssh-restrict#3]
We failed to define a command with uppercase letters in the name.

this config won't work:

```
[commands]
/usr/bin/AWESOMEcommand = /usr/bin/AWESOMEcommand
```

Reading https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser there is this statement:

> All option names are passed through the optionxform() method.  Its default implementation converts option names to lower case.

**Comment by mxey** [comment]
Apparently not that hard to fix?
https://docs.python.org/2/library/configparser.html#ConfigParser.RawConfigParser.optionxform

Would you submit a pull request?

**Comment by vinzent** [comment]
fixed by #4
